### ISSUE TITLE
FADE_HORIZONTAL not exported

### ISSUE THREAD

**Issue by writzx** [Traviskn/react-router-native-stack#40]
The error that I get.

```
./node_modules/react-router-native-stack/lib/getDimension.js
Attempted import error: 'FADE_HORIZONTAL' is not exported from './animationTypes'.
```

Easily fixed by adding the line `export const FADE_HORIZONTAL = 'fade-horizontal';` to the error file [have to do this every run]

Not sure if its a bug. I am pretty new to node and react.

And adding a small bit of info if that matters: I am using `react-native-web`.

**Comment by anthonykrivonos** [comment]
For those people still getting this issue, I submitted a PR for it: #42

**Comment by Traviskn** [comment]
resolved by #42
